content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+individuals, I work on species as evolutionary lineages.
+A good way to show the evolutionary relationships between species
+are phylogenies. For example, this a phylogeny, with four species,
+eloquently named '1', '2', '3' and '4':
+
+![A phylogeny with four species](img/blog-images/2019-01-06-babette/phylogeny.png)
+
+In my research, I simulate thousands of these. I call these 
+phylogenies 'the truth'. Then I wonder, would such a phylogeny actually
+be true, how well would we be able to infer it? I say 'infer', instead
+of 'measure', as we cannot measure phylogenies directly in nature. 
+
+In nature, however, we can sequence the DNA of species. From these DNA
+sequences, we use software to neatly arrange these sequences, creating
+a DNA alignment. Such DNA alignments are where most researchers start from.
+Here is such an alignment, of four (eloquently named) species
+and their aligned DNA sequences, in which a different color represents a
+different nucleotide:
+
+![A DNA alignment of four species](img/blog-images/2019-01-06-babette/alignment.png)
+
+It is hard to only guess by eye which of the four species are most related.
+Luckily, we have software for such problems, for example, BEAST2.
+BEAST2 is a Bayesian phylogenetic tool that infers a posterior
+from a DNA alignment. The posterior contains multiple phylogenies,
+in which those that are more likely are present more often. Here is what these
+posterior trees look like when superimposed:
+
+![Posterior trees of four species](img/blog-images/2019-01-06-babette/densitree.png)
+
+Spoiler: actually, the phylogeny shown was the true phylogeny 
+underlying the alignment! You can see the inference has
+problems recovering it. Part of my research is to measure
+exactly this error. Therefore, I would not only need
+thousands of phylogenies, but also thousands of posteriors.
+There was a practical reason to make this go smooth!
+
+#### Before babette
+
+Before `babette` existed, it would take three steps to plot
+the phylogenies in a posterior:
+
+Step|      Input      |   Tool  |        Output
+----|-----------------|---------|-----------------------
+   1|alignment        |BEAUti   |BEAST2 input file
+   2|BEAST2 input file|BEAST2   |posterior files
+   3|posterior files  |DensiTree|plot of the phylogenies
+
+ 1. Run BEAUti to create a BEAST2 input file
+ 2. Run BEAST2 to create a posterior
+ 3. Run DensiTree to plot the phylogenies in a posterior
+
+BEAUti (I assume the pun to BEAST is intended) is a beginner-friendly program 
+with a graphical user interface. To analyse an alignment such as above,
+one needs at least 7 mouse clicks or 3 keyboard commands to create
+a BEAST2 input file. BEAST2 can be called from the command line to
+read the input file and create a posterior saved as multiple files.
+DensiTree is another beginner-friendly program with a graphical
+user interface to load the phylogenies of a posterior and plot these. 
+
+Again, I would need thousands of these posteriors.
```

Comment:
Can you extrapolate to number of mouse clicks too?

(not required)

Reply:
Added :+1: